### PR TITLE
Update Click dependency

### PR DESCRIPTION
**Pull request by Chickensoupwithrice**
allow the use of newer versions of click

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
       license='BSD-2-Clause',
       keywords='cli vim vimwiki wiki',
       install_requires=[
-          'click~=7.1',
```
Comment (sstallion) [inline setup.py:37]: Hi @Chickensoupwithrice thanks for the pull request! What would you think of changing this line to `'click>=7.1',`?

Comment (Chickensoupwithrice) [reply setup.py:37]: yeah that works for me :) gimme a bit will update

Comment (Chickensoupwithrice) [reply setup.py:37]: Done :3

**Comment by sstallion** [comment]
Thanks! I'm out on holiday this weekend but I should be able to get a new release made next week.

**Pull request merged**